README.md

```diff
@@ -15,6 +15,12 @@ A linter tool to normalize `.xib` and `.storyboard` files. Inspired by [realm/Sw
 $ brew install IBDecodable/homebrew-tap/iblinter
 ```
 
+### Using [Mint](https://github.com/yonaskolb/Mint)
+
+```sh
+$ mint install IBDecodable/IBLinter
```

Comment:
Could you add executable name `iblinter` because there are multiple executable target.

```sh
$ mint run IBDecodable/IBLinter
🌱  Finding latest version of IBLinter
🌱  IBLinter 0.4.20 already installed
There are multiple executables, which one would you like to run?

You must respond with one of the following:
iblinter
iblinter-tools
```

Reply:
I tended just to run `iblinter` without `mint run ...` as `/usr/local/bin` is in my path.

However, I can clarify this a little.

Reply:
Ah, sorry I misunderstood, you used `install` instead `run`, I see.